examples/wrapped/config/baroclinic.yml

```
@@ -1,6 +1,6 @@
 data_table: default
-diag_table: /fv3core/comparison/wrapped/config/diag_table_baroclinic
-field_table: gs://vcm-fv3config/config/field_table/TKE-EDMF/v1.0/field_table
+diag_table: gs://vcm-fv3config/config/diag_table/baroclinic_test/v1.0/diag_table
+field_table: gs://vcm-fv3config/config/field_table/baroclinic_test/v1.0/field_table
 experiment_name: baroclinicnew
```

Try to avoid `new` in an experiment name, it will eventually not be.
```suggestion
experiment_name: baroclinic
```